backend/core/serializers.py

```
@@ -9,7 +9,7 @@
 from core.models import *
```

Flake8 (7.2.0)

[error] 'from core.models import *' used; unable to detect undefined names

(F403)